packages/ember/index.js

```diff
@@ -27,10 +27,20 @@ module.exports = {
     },
   },
 
-  config(_, appConfig) {
-    const addonConfig = appConfig['@sentry/ember'];
-    this.options['@embroider/macros'].setOwnConfig.sentryConfig = { ...addonConfig };
-    return this._super(...arguments);
+  included() {
+    const app = this._findHost();
+    const config = app.project.config(app.env);
+    const addonConfig = config['@sentry/ember'] || {};
+
+    if (!isSerializable(addonConfig)) {
+      console.warn(
+        "Warning: You passed a non-serializable config to `ENV['@sentry/ember'].sentry`. Non-serializable config (e.g. RegExp, ...) can only be passed directly to `Sentry.init()`.",
```

Comment:
Do you think it'd be worth explaining that it's `Sentry.init()` inside their app to explain both where it goes and why non-serializable config works there?